src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machineArray = {new Truck(), new Bulldozer(), new Excavator()};
 
```

Comment:
Redundant empty line

Reply:
Noted! Next time I will be more careful.